README.md

```diff
@@ -61,6 +61,23 @@ to test your that your setup is working.
 
 Note: if you use multiple Kubernetes clusters, you can switch to Minikube with `kubectl config use-context minikube`
 
+### Select the namespace to deploy to
+
+The operator can run in a namespace other than `default`. For example, to deploy it to the `test` namespace, run the following:
```

Comment:
Minor nitpick, but I'd say "run the following before deploying the operator's manifests" or somesuch, as just running those commands on their own does not deploy the operator to the `test` namespace as stated.